index.js

```diff
@@ -0,0 +1,33 @@
+// 1. console log a line of text
+console.log("This is the first exercise");
+
```

Comment:
The content of console.log should be "this is an exercise in console logging" :))